cluster/examples/kubernetes/ceph/pool.yaml

```
@@ -54,6 +54,11 @@ spec:
     mirror:
       disabled: false
       interval: 60s
+  # quota in bytes and/or objects, default value is 0 (unlimited)
+  # see https://docs.ceph.com/en/latest/rados/operations/pools/#set-pool-quotas
+  quotas: {}
```

How about commenting it out instead of adding the `{}`?
```suggestion
  # quotas:
```

---

Done. I also tested uncommenting quotas,maxBytes and MaxObjects but not entering any values. In that case maxBytes and MaxObjects will be nil and no changes to quotas are made so this does seem to be a robust solution.